app/components/mandaat/burgemeester-selector.js

```diff
@@ -41,7 +41,10 @@ export default class MandaatBurgemeesterSelectorComponent extends Component {
 
   async load() {
     await this.loadBurgemeesterMandates();
-    this.persoon = await this.loadBurgemeesterPersoon();
+
+    if ((this.burgemeesterMandate && this, this.voorzitterVastBureauMandate)) {
```

Comment:
But weird syntax, I would just use 
`if (this.burgemeesterMandate && this.voorzitterVastBureauMandate) {`
if that is possible

Reply:
This is going to be added when selecting a suggestion I think, I was lucky it didn't break it ;p